doc/Caliptra.md

```diff
@@ -931,22 +938,86 @@ A detailed description of the POST and CAST KATs can be found at csrc.nist.gov.
 
 As shown in *Table 15: POST/CAST usage*, since the cryptographic algorithms required by the Caliptra Boot ROM are considered POSTs, and those same algorithms are used by Caliptra FMC and FW, there is no requirement that FMC and Runtime FW implement CASTs for those algorithms.
 
-## FW Signing/Verification algorithms
-
-Caliptra firmware is composed of multiple layers: an FMC and an application firmware image. Each layer is signed individually by ECDSA P384 keys, and optionally [LMS Signatures](#post-quantum-cryptography-pqc-requirements) as well.
-
-Each layer is signed by a vendor-controlled key. In addition, each layer may also be signed by an owner-controlled key. The image header contains both the owner public key and the signature using that key.
-
-During boot, Caliptra ROM shall verify the vendor signature over FMC before allowing that FMC to run.
-
-See the [Owner authorization](#owner-authorization) section for how the owner key is used.
-
-Caliptra's FW signature generation and verification shall follow the requirements described in [Reference 3](#ref-3).
+## Firmware image format and verification
+
+Caliptra supports verifying firmware with ECDSA P384 signatures and [Leighton-Micali Hash-based Signatures (LMS)](#post-quantum-cryptography-pqc-requirements) in accordance with the requirements described in [Reference 3](#ref-3).
+
+Caliptra firmware is composed of two images: an FMC image and an application firmware image. A single firmware manifest describes these images. The manifest consists of a preamble (*Table 16*), a header (*Table 17*), and a table of contents (*Table 18*). The image layout is shown in *Figure 9: firmware image layout*.
+
+![](./images/fw-img-bundle.svg)
+
+*Figure 9: Firmware image lyaout*
+
+To verify the firmware, Caliptra ROM performs the following steps:
```

Comment:
Any reason to use double negative throughout this process? steps 1, 2, 10, and 12.

Example from step 12: 

it might be clearer instead of: 
`lifecycle is not unprovisioned or anti-rollback is not disabled`
to say:
`lifecycle is provisioned or anti-rollback is enabled`

Reply:
yes the reason for the double negative is because "unprovisioned" is an explicit lifecycle state defined in the security state section. Same for anti-rollback where there is an explicit fuse called "anti-rollback disable". Would quotes help?

Reply:
I think quotes help. So it could be: `lifecycle is not in "unprovisioned" or "anti-rollback disable" state,...`?